src/App.vue

```diff
@@ -1,4 +1,15 @@
 <template>
+  <teleport to="head">
+    <meta property="title" :content="metadata.title">
+    <meta property="description" :content="metadata.description">
+    <meta property="og:title" :content="metadata.title">
+    <meta property="og:description" :content="metadata.description">
+    <meta property="og:image" content="https://near-in-minutes.com/img/logo.dbccf5a9.png" />
```

Comment:
@oceanByte is this `img` url consistent?  seems build-dependent to me given the value injected into the filename `logo.png`

Reply:
Yes, my bad, I was using it for testing and forget to update it properly. 

Quickest solution: I can create a public folder for static files. 

In the best case though you'd have a CDN like CloudFront that takes care of image distribution to avoid any unnecessary requests by other servers to get the image file from our server. Is there something like this already in place?

Reply:
Nothing like this in place but I think render.com handles some of these issues for us already, don't they?

Reply:
I don't know how renderer.com handles this but it should be not critical if we do not expect thousands of users at any given point in time.

I added a preview image to the public folder and updated the path.